src/main/resources/log4j2.properties

```diff
@@ -1,4 +1,4 @@
-rootLogger.level = error
+rootLogger.level = info
```

Comment:
should we default to warn? Info is really verbose

Reply:
Netty renders this at Info level - 

INFO  org.springframework.boot.web.embedded.netty.NettyWebServer.start (89) - Netty started on port(s): 4120

If we change this to warn, we would not see the Netty started message